contents/handbook/people/team-structure/product-analytics/objectives.mdx

```diff
@@ -1,72 +1,54 @@
-
-</HideFromJobPosting>
\ No newline at end of file
+### Objective 1: Nail Data Exploration
+
+1. Build v0.1 of the data exploration system
+2. Run a design sprint with Cory
+3. Ship v2
+
+#### Why?
+
+Product reason:
+
+- We have a pretty good interface for data *visualization* (trends, funnels, retention, etc), but not so much for data 
+  *exploration* (events, persons). Users have been asking for more.
+- The biggest pain comes from not being able to move between different types of related data. Imagine [this scenario](https://github.com/PostHog/posthog/issues/7963#issuecomment-1010235411) (© @rcmarron)
+> 0) Found an anomaly about feature usage in trends
+> 1) "I wonder which paying organizations have used feature Y"
+> 2) Search for that event/action and change the tab to organizations
+> 3) "Interesting CompanyXYZ is using it, I wonder who"
+> 4) Flip to persons + add an organization filter for CompanyXYZ
+> 5) See the persons
+> 6) "I wonder how/why they're using it"
+> 7) Click recordings + see the recordings
+> It would be pretty clunky/impossible to explore your data in that way with today's interface. 
+- Currently you need to re-create this same filtered query on 3-4 different pages and manually copy over the properties. 
+  It's doable, but not a great experience. 
+
+Code reason:
+- We have a lot of bespoke bridges to show related data in the interface (people on insights, recordings on people).
+- Every new one adds more complexity. 
+- This compounded by ever so slightly different use of filters on different data types.
+- The further we continue down this path, the harder it'll be to have a fully integrated product across all data.
+
+Meta reason:
+- We now have separate product teams. 
+- If we're not careful, we'll end up with inoperable products. 
+- This builds the first shared bridge between all of them.
+
+#### Potential ideas for how?
+- Universal Search: https://github.com/PostHog/posthog/issues/7963 (Note: this doesn't mean natural language text search)
+- An "explore" interface/table, which supports input from various swappable data sources, with a unified set of filters applied to data.
+- Support for custom columns and [formulas](https://github.com/PostHog/posthog/issues/11913) on those tables.
+- Ability to save as a cohorts, save as an action, or export as a CSV when the data permits.
+- The ability to link to these tables from anywhere. "Explore further" links wherever we show *any* data in the interface.
+
+#### Future steps.
+- Replace all other data tables in the app (events list, persons list, etc) with this interface.
+- Interconnect all the ideas here: https://github.com/PostHog/posthog/issues/3765
+- Add this power to the toolbar. E.g. add an action, see user recordings and explore further.
+
+### Objective 2: Support performant querying of every insight type at a scale of billions scale events
```

Comment:
@mariusandra probably obvious, but I presume motivation for objective 2 is "to avoid a poor experience for our largest scale customers" ?

Reply:
Is it possible to shift the focus of the key results to outputs that stem from the various tasks listed here? How will you measure the success of these activities?